packages/cubejs-snowflake-driver/package.json

```diff
@@ -19,7 +19,7 @@
   "dependencies": {
     "@cubejs-backend/query-orchestrator": "^0.27.5",
     "date-fns-timezone": "^0.1.4",
-    "snowflake-sdk": "^1.1.15"
+    "snowflake-sdk": "1.6.1"
```

Comment:
It's better to use `^1.6.1`, because `snowflake-sdk` can ship feature updates.

Reply:
Done ✅